src/components/SideBar.jsx

```diff
@@ -0,0 +1,54 @@
+import { Flex } from "@sparrowengg/twigs-react";
+
+import { FaShoppingCart } from "react-icons/fa";
+import { IoHome } from "react-icons/io5";
+
+import { FaSignOutAlt } from "react-icons/fa";
+import { useNavigate } from "react-router-dom";
+const SideBar = () => {
+  const navigate = useNavigate();
+  const handleLogOut = () => {};
+  return (
+    <>
+      <Flex
+        css={{ background: "white" }}
+        flexDirection="column"
+        justifyContent="space-between"
+        alignItems="center"
+      >
+        <Flex>
+          <img
+            src="https://asset.brandfetch.io/id4QM--uk4/idZgawl2X6.png"
+            width={"50px"}
+            height={"50px"}
+            alt=""
+          />
+        </Flex>
+        <Flex flexDirection="column" css={{ gap: "$20" }}>
+          <Flex>
+            <IoHome
+              onClick={() => navigate("/home")}
+              className="icon-hover"
```

Comment:
@arun-121 is this class name needed if you want to achieve hover functionality in twigs you can do like this
```<Flex css={{
"&:hover": { background: 'red' }
}}```